opdracht1.js

```diff
@@ -0,0 +1,18 @@
+const tvTypes = inventory.map((tvType) => {
+    return tvType.type;
+})
+console.log(tvTypes);
+
+const tvsSoldOut = inventory.filter((tvSoldOut) => {
+    if (tvSoldOut.sold - tvSoldOut.originalStock === 0)
+        return tvSoldOut;
+})
```

Comment:
De filter functie verwacht een eigenlijk ten alle tijden een boolean terug te krijgen. Uiteindelijk zal `undefined` vertaald worden naar `false`, maar het is toch altijd goed om gewoon een boolean terug te geven, omdat dat expliciet is
```suggestion
const tvsSoldOut = inventory.filter((tvSoldOut) => {
    return tvSoldOut.sold === tvSoldOut.originalStock
})
```